test/test_commit.py

```diff
@@ -149,6 +149,81 @@ def check_entries(d):
+     :start-after: # [25-test_references_and_objects]
+     :end-before: # ![25-test_references_and_objects]
+ 
+-You can easily access configuration information for a remote by accessing options as if they where attributes. The modification of remote configuration is more explicit though.
++You can easily access configuration information for a remote by accessing options as if they were attributes. The modification of remote configuration is more explicit though.
+ 
+ .. literalinclude:: ../../test/test_docs.py
+     :language: python
+@@ -391,7 +391,7 @@ Here's an example executable that can be used in place of the `ssh_executable` a
+     ID_RSA=/var/lib/openshift/5562b947ecdd5ce939000038/app-deployments/id_rsa
+     exec /usr/bin/ssh -o StrictHostKeyChecking=no -i $ID_RSA "$@"
+ 
+-Please note that the script must be executable (i.e. `chomd +x script.sh`). `StrictHostKeyChecking=no` is used to avoid prompts asking to save the hosts key to `~/.ssh/known_hosts`, which happens in case you run this as daemon.
++Please note that the script must be executable (i.e. `chmod +x script.sh`). `StrictHostKeyChecking=no` is used to avoid prompts asking to save the hosts key to `~/.ssh/known_hosts`, which happens in case you run this as daemon.
+ 
+ You might also have a look at `Git.update_environment(...)` in case you want to setup a changed environment more permanently.
+ 
+@@ -509,14 +509,14 @@ The type of the database determines certain performance characteristics, such as
+ 
+ GitDB
+ =====
+-The GitDB is a pure-python implementation of the git object database. It is the default database to use in GitPython 0.3. Its uses less memory when handling huge files, but will be 2 to 5 times slower when extracting large quantities small of objects from densely packed repositories::
++The GitDB is a pure-python implementation of the git object database. It is the default database to use in GitPython 0.3. It uses less memory when handling huge files, but will be 2 to 5 times slower when extracting large quantities of small objects from densely packed repositories::
+ 
+     repo = Repo("path/to/repo", odbt=GitDB)
+ 
+ 
+ GitCmdObjectDB
+ ==============
+-The git command database uses persistent git-cat-file instances to read repository information. These operate very fast under all conditions, but will consume additional memory for the process itself. When extracting large files, memory usage will be much higher than the one of the ``GitDB``::
++The git command database uses persistent git-cat-file instances to read repository information. These operate very fast under all conditions, but will consume additional memory for the process itself. When extracting large files, memory usage will be much higher than ``GitDB``::
+ 
+     repo = Repo("path/to/repo", odbt=GitCmdObjectDB)
+ """
+        commit = self.rorepo.commit('c0740570b31f0f0fe499bf4fc5abbf89feb1757d')
+        patch = commit.patch
+        self.assertEqual(patch[:200], expected_path_result[:200])
```

Comment:
I don't understand why it would slice the result into lines like this. This also means it would possibly only test for equivalence on a subset of lines, missing some lines behind the 600 mark, for example.

Reply:
I can't test with whole patch text. I have to set maxDiff = None to disable limit and test. For test_with_parent, I will try to get a commit with smaller patch. For test_with_no_parent, which is first commit, I will save patch text to file and set maxDiff in setup and test it.

Reply:
Then I think it would be important to make clear how `maxDiff` works in the property's documentation so callers know how to deal with it. To me for instance it wasn't clear this exists.

Or said differently, if there is a technical limitation there is less of a need to work around it from my side, as long as it can be made clear in the docs how to handle it.